templates/js/custom-copilot-assistant-assistants-api/.vscode/launch.json.tpl

```diff
@@ -75,6 +75,30 @@
                 "order": 3
             },
             "internalConsoleOptions": "neverOpen",
+        {{#CEAEnabled}}
+        },
+        {
+            "name": "Launch Remote in Copilot (Edge)",
+            "type": "msedge",
+            "request": "launch",
+            "url": "https://www.office.com/chat",
```

Comment:
Add account hint? See https://github.com/OfficeDev/teams-toolkit/blob/dev/templates/js/sso-tab-with-obo-flow/.vscode/launch.json#L63. 
You can test if the account hint is workable.

Reply:
ok, added